doc/nrf/libraries/nrf_security/doc/driver_config.rst

```diff
@@ -302,7 +302,9 @@ You can use the following Kconfig options for fine-grained control over which dr
 RNG configurations
 ******************
 
-To enable PRNG seeded by entropy (also known as TRNG), set one or more of the following configurations:
+To enable RNG, set the following configuration: :kconfig:option:`CONFIG_PSA_WANT_GENERATE_RANDOM`
+
+RNG uses a PRNG seeded by entropy (also known as TRNG), when RNG is enabled set at least one of the following configurations:
```

Comment:
It is more logical to say exactly one here, right?
I will change this so that it says that you can only select one later but it is better to have it from now I think.

Reply:
At the moment you can select both. There is a note below that states this:
>  If multiple PRNG algorithms are enabled at the same time, CTR_DRBG will be prioritized for random number generation through the front-end APIs for PSA Crypto